src/module/system/settings/index.ts

```diff
@@ -343,6 +343,14 @@ function registerTrackingSettings(): void {
         },
     });
 
+    // Tracks the last party folder state for next launch. Defaults to true so that "No Members" shows on initial creation.
+    game.settings.register("pf2e", "activePartyFolderState", {
+        name: "Active Party Opened or closed",
+        config: false,
```

Comment:
```suggestion
        scope: client,
        config: false,
```

It's the default, but good to be explicit.

Reply:
Indeed it is, because I forgot it for active party, and its where I copied from. I suppose this is now also a bug fix